src/test/java/seedu/cafectrl/command/ListIngredientCommandTest.java

```diff
@@ -37,7 +37,7 @@ public void showToUser(String... message) {
         listIngredientCommand.execute(menu, ui);
 
         String expectedOutput = "Chicken Rice Ingredients: \n" +
```

Comment:
for this variable, perhaps have the plus operator in the next line, according to coding standards the break should be before the operator, same for line 40